Release notes for 9.4.1 (#19096)
elastic/logstash: docs/release-notes/index.md

% ### Fixes [logstash-next-fixes]
% *

## 9.4.1 [logstash-9.4.1-release-notes]

### Features and enhancements [logstash-9.4.1-features-enhancements]

#### Dead letter queue flush check interval [logstash-9.4.1-dlq-flush-check-interval]

Introduces new `dead_letter_queue.flush_check_interval` config for flushing the staled segment files scheduler 
which can reduce frequent check overhead.

> If you are using intensive DLQ operations (write/read), the frequent flush check scheduler might give overhead to the pipeline, 

Reviewer: ```suggestion
If you are using intensive DLQ operations (write/read), the frequent flush check scheduler might create more overhead for the pipeline, increasing CPU usage. 
```

> means uses much CPU. Introducing configurable scheduler cadence improves the pipeline efficiency by removing frequent operations 

Reviewer: ```suggestion
Introducing configurable scheduler cadence improves the pipeline efficiency by removing frequent operations 
```

[#19036](https://github.com/elastic/logstash/pull/19036).

### Updates to dependencies [logstash-9.4.1-dependencies]

* Update bundled JDK to 21.0.11 build 10.

### Plugins [logstash-plugin-9.4.1-changes]

**Beats Input - 7.0.9**

* Update Netty dependency to 4.1.133.Final [#539](https://github.com/logstash-plugins/logstash-input-beats/pull/539)

> 

Reviewer: ```suggestion
```
